### DISCUSSION TITLE
Add registration module

### DISCUSSION THREAD

**Discussion by infnibor** [pelican-dev/panel#808] [Ideas]
Introduce the registration module directly under the login panel

**Comment by gOOvER** [comment]
in my eyes not needed. Only for Hosters. That would sth for a Plugin, but not in Pelican itself

**Comment by notAreYouScared** [comment]
We have no interest in adding a self-registration feature at this time. Thanks!

**Comment by Boy132** [comment]
Duplicate of https://github.com/pelican-dev/panel/discussions/17
